src/core/component.js

```diff
@@ -203,6 +203,7 @@ Component.prototype = {
     if (!this.initialized) {
       this.init();
       this.initialized = true;
+      if (el.isPlaying) { this.play(); }
```

Comment:
Can you rephrase? What's the confusing part? What the line does it's to call the play method on the component if the corresponding entity is playing.